### ISSUE TITLE
reveal.JS export seems bogus

### ISSUE THREAD

**Issue by pagiraud** [Zettlr/Zettlr#3770]
### Description

Hello and thank you for this great piece of software.

Well, but if I’m doing everything wrong, the reveal.JS export seems bogus or unclear.

In the assets manager, the Reveal.js export (different typography that what you can find clicking the export button at the top of the window, where you can read reveal.JS), it looks like you can use 

```
include-before-body: []
include-after-body: []
include-in-header: []
```
to inject code, but actually you can’t since according to the Pandoc documentation these directives can only be used with standalone (which is set to false in Zettlr with a warning saying it must stay this way).

In the pandoc documentation, it is not said tha the `css: `directive must be used with the standalone option but in fact it must (I made tests).

So, to sum up, there are many ways for the Pandoc user to inject code in his or her presentation, but in Zettlr you have to put your css directly in your document. It doesn’t seem very elegant to me. I really would like to be able to use `css: ` in the YAML frontmatter.

At least, IMHO, there should be some warning that these directives can’t be used.

I’ve written it seems bogus but maybe you will consider this is only an enhancement proposal.

Cheers,

### Reproducing

.

### Zettlr Version

Stable (most recent version)

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Kubuntu 22.04

**Comment by nathanlesage** [comment]
> to inject code, but actually you can’t since according to the Pandoc documentation these directives can only be used with standalone (which is set to false in Zettlr with a warning saying it must stay this way).

For the upcoming 3.0 release I completely removed that weird stuff Zettlr was pulling off with RevealJS presentations, so they should be able to be used just like any other export format would be. Feel free to double check in the current nightlies, these should already work this way.
